log/default.go

```diff
@@ -0,0 +1,89 @@
+package log
+
+import (
+	"encoding/json"
+
+	"github.com/newrelic/newrelic-telemetry-sdk-go/telemetry"
+	iLog "istio.io/pkg/log"
+)
+
+const (
+	// AdapterScopeName is the default logging scope name for this adapter.
+	AdapterScopeName       = "newrelic"
+	defaultOutputLevel     = ErrorLevel
```

Comment:
Since `error` is the default output we should update this line in the DEVELOPMENT.md
```
The newrelic-istio-adapter should output activity to STDOUT signifying that metrics have been captured and sent to New Relic.
```
No output is expected now if things are working correctly

Reply:
Should also clarify this line in the README.md
```
The logs of Mixer and the newrelic-istio-adapter in Kubernetes should show activity or errors.
```

Reply:
Good catches :+1: